app/code/Magento/Store/Model/StoreRepository.php

```diff
@@ -67,13 +67,15 @@ public function get($code)
         }
 
         $storeData = $this->getAppConfig()->get('scopes', "stores/$code", []);
-        $store = $this->storeFactory->create([
-            'data' => $storeData
-        ]);
+        $store = $this->storeFactory->create(
+            [
+                'data' => $storeData
+            ]
+        );
 
         if ($store->getId() === null) {
             throw new NoSuchEntityException(
-                __("The store that was requested wasn't found. Verify the store and try again.")
+                __("The store with code $code that was requested wasn't found. Verify the store and try again.")
```

Comment:
@avoelkl Does this error message appear in the browser (admin and/or website)? If yes, can you please escape the output of `$code`.

Reply:
Hi, actually I updated this message while i was working on the CLI but it would be possible that this message is shown in a browser as well (in developer mode). However, the store code is no user specific output. The code comes from a config.php file or from the database. Do you think we need to escape this? Any recommendations on what to use here?

Reply:
@avoelkl Thanks for checking, I believe it should be fine then. Can you make the error message translation-safe like Pete mentioned?